lib/fastlane/plugin/fueled/actions/upload_to_test_fairy_android.rb

```diff
@@ -0,0 +1,76 @@
+require_relative '../helper/fueled_helper'
+
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class UploadToTestFairyAndroidAction < Action
+      def self.run(params)
+        if other_action.is_ci
+          other_action.testfairy(
+            api_key: params[:api_key],
+            apk: params[:apk],
```

Comment:
@samuelskelton : By default it picks the aab file so will this works? i.e. we need to mention `BUILD_FORMAT=apk` in env file right? _(Correct me if i'm wrong :see_no_evil:)_

Reply:
You're right. So I'm wondering if I should remove the default for Android?

In the MeowWolf code I'm calling for Fastlane to build an APK specifically, and then I pass in the APK path on init:
```
   upload_to_test_fairy_android(
      api_key: ENV['TESTFAIRY_ACCESS_KEY'],
      apk: "./build/app/outputs/flutter-apk/app-release.apk",
    )
```
So it works fine because I'm passing the path, but do you have a suggestion on what the default should/shouldn't be for this?

Reply:
I think we should configure cases like `aab`, `apk`, or `dual`, in the case of `dual` it will export both types we can pick either both or one of them. Then we can use where ever as per requiremens.

Reply:
I think @shkcodes was saying that in the case of something like building for AppCenter (where it does both), it does it in such a way that we can't really get at it.

Otherwise, there isn't a single command that will output both. They are separate commands and it will increase the build/processing time. I think in this case it's best to leave as-is, and allow the user to choose which format they need right from the beginning 👍 @thapli9A